libs/app-shell/src/lib/api/events/messaging.service.spec.ts

```diff
@@ -139,13 +139,13 @@ describe('MessagingService', () => {
                     eventType: EventType.DEFAULT, shared: true, prefix: 'system:'
                 });
                 let expectedMessage = 'none';
-                service.onMessage('system:events', (msg: Message) => {
+                service.subscribe('system:events', (msg: Message) => {
```

Comment:
I had Subscribe before, and I changed to onMessage as I though its more intuitive. If you think subscribe feels better we will leave this

Reply:
Angular devs, subscribe/unsubscribe/emit are native words now ;)